docs/reference/docs/update.asciidoc

```diff
@@ -109,22 +109,22 @@ curl -XPOST 'localhost:9200/test/type1/1/_update' -d '{
 If both `doc` and `script` is specified, then `doc` is ignored. Best is
 to put your field pairs of the partial document in the script itself.
 
-By default if `doc` is specified then the document is always updated even
-if the merging process doesn't cause any changes.  Specifying `detect_noop`
-as `true` will cause Elasticsearch to check if there are changes and, if
-there aren't, turn the update request into a noop. For example:
+By default if `doc` is specified then the document only updated even if the
```

Comment:
"If `doc` is specified, its value is merged with the existing `_source`. By default, the document is only reindexed if the new `_source` field differs from the old.  Setting `detect_noop` to `false` will cause Elasticsearch..."